docs/tutorials/kernel-density.md

```diff
@@ -0,0 +1,265 @@
+Kernel Density
+--------------
+
+This document provides a detailed example on how to build a raster from point data using kernel density estimation.  Though that is the ostensible point, it also provides a brief introduction to working with rasters, including how to tile a raster and how to use the result as the basis for a computation in Spark.
+
+Kernel density is one way to convert a set of points (an instance of vector data) into a raster.  In this process, at every point in the point set, the contents of what is effectively a small Tile (called a Kernel) containing a predefined pattern are added to the grid cells surrounding the point in question (i.e., the kernel is centered on the tile cell containing the point and then added to the Tile).  This is an example of a local map algebra operation.  Assuming that the points were sampled according to a probability density function, and if the kernel is derived from a Gaussian function, this can develop a smooth approximation to the density function that the points were sampled from.  (Alternatively, each point can be given a weight, and the kernel values can be scaled by that weight before being applied to the tile, which we will do below.)
+
+To begin, let's generate a random collection of points with weights in the range (0, 32).
+
+```scala
+  import geotrellis.vector._
+  import scala.util._
+
+  val extent = Extent.fromString("-109, 37, -102, 41") // Extent of Colorado
```

Comment:
This may be the first place a new user ever sees the `Extent` class used. Is it the same to do:

```scala
val extent = Extent(-109, 37, -102, 41)
```
?